crates/iroha_genesis/src/lib.rs

```diff
@@ -356,6 +389,53 @@ impl Decode for ExecutorPath {
     }
 }
 
+/// For readability of [`RawGenesisTransaction`], briefly denotes a trigger with wasm executable registered in genesis
+#[derive(Debug, Clone, Serialize, Deserialize, IntoSchema, Encode, Decode, Constructor)]
+pub struct GenesisWasmTrigger {
+    id: TriggerId,
+    action: GenesisWasmAction,
+}
+
+/// For readability of [`GenesisWasmTrigger`], briefly denotes an action with wasm executable
+#[derive(Debug, Clone, Serialize, Deserialize, IntoSchema, Encode, Decode)]
+pub struct GenesisWasmAction {
+    /// Expected to be converted by [`iroha_test_samples::load_sample_wasm`]
+    executable: String,
```

Comment:
```suggestion
    executable: PathBuf,
```

like `ExecutorPath`

Reply:
Changed `ExecutorPath` to `WasmPath` and shared it